apps/jira-mapping/.env

```diff
@@ -0,0 +1,9 @@
+PORT=8302
+MONDAY_SIGNING_SECRET=
+TUNNEL_SUBDOMAIN=jql-app-803
```

Comment:
This one will not work for other app ids, we need to keep it blank for the templates to work correctly